### PR TITLE
Fix issues 27 and 30

### PR DESCRIPTION
**Pull request by jwhitt3r**
This PR addresses the issues outline in Fix readme https://github.com/trypear/pearai-app/issues/27

I have reworked the page to link to the master document, included the contributing section to align with the contents of the document.

Updated the features to be inline with the webpage
Added a roadmap linking the master document
Added a license section linking to your license model, I have removed the MIT license at the top as that is confusing with the actual document
Added a contact section
Added an acknowledgement section listing contributors
Reworded the opening paragraph
Reworked the page to include the Prerequisites of an OpenAI API key

This relinks the work in PR https://github.com/trypear/pearai-app/pull/30

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,77 +1,72 @@
 # Pear: The Best AI-Powered Code Editor
 
-Supercharge your development in an editor designed for less coding, with AI. This repository is a fork of VSCode.
+Speed up your development with an editor integrated with AI. PearAI is forked from VSCode, and aims to reduce the time from ideation to conception for your product development by achieving the most seamless integration with AI. By developers, for developers.
```
Comment (nang-dev) [inline README.md:3]:
```suggestion
Supercharge your development with an editor designed for less coding, using AI. PearAI is forked from VSCode, and aims to reduce the time from ideation to conception for your product development by achieving the most seamless integration with AI.
```

**Review by nang-dev — COMMENTED** [review]
Looks great! just a small thing <3

File: README.md (hunk 1 above)
Comment (jwhitt3r) [reply README.md]: Made the change

**Review by nang-dev — APPROVED** [review]

**Comment by jwhitt3r** [comment]
Thanks for the merge, that was a bit of a journey!

**Pull request merged**